### ISSUE TITLE
Monospaced font does not use the default one?

### ISSUE THREAD

**Issue by virtuallyvlad** [MaxLap/standard-notes-indent-editor#22]
On my Standard Notes app on Windows, without any theme installed, the Plain Editor uses Consolas 11 as the font.  But if switching to the Indent Editor and enabling the monospace font, the font becomes some other bigger, wider font (maybe Courier New, 12?).

Personally I prefer Consolas 11, but at the very least this is inconsistent with the Standard Notes default.

**Comment by MaxLap** [comment]
You're right, I hadn't noticed. I'll see if I can make it fit better, which is not at all clear to me. Documentation for extension used to be quite bad.

**Comment by MaxLap** [comment]
I looked into it, didn't find how a custom extension is meant to be themed. I don't have any desire to spend more time on this editor, so sorry but unless you submit a PR to fix this, I doubt this is get fixed.

**Comment by virtuallyvlad** [comment]
> I looked into it, didn't find how a custom extension is meant to be themed. I don't have any desire to spend more time on this editor, so sorry but unless you submit a PR to fix this, I doubt this is get fixed.

Thanks for checking.  And just curious -- are you still using this editor or have you moved onto something else?

**Comment by MaxLap** [comment]
This is the only editor I use. I absolutely hate working in JS and yet I spent days doing so to make it.

**Comment by virtuallyvlad** [comment]
Ah ok, so I'm guessing it works well for your purposes.  I'm assuming other quirks/issues would have to be somewhat major to merit your attention, so will hold off for the time being.  Thanks for your efforts suffering with JS; it's appreciated.
